docs/developer/extending/apps/architecture/communication-between-app-and-saleor.mdx

```diff
@@ -1,3 +1,142 @@
 ---
 title: Communication between App and Saleor
 ---
+
+## Saleor Asynchronous Events
```

Comment:
I think you can use h1?

Reply:
Should be h2. @peelar suggested using it in the docs (metadata docs use it for example)